src/core/algorithms/statistics/data_stats.h

```diff
@@ -120,10 +130,34 @@ class DataStats : public Algorithm {
     Statistic GetNumberOfLowercaseChars(size_t index) const;
     // Returns number of digit chars in a string column.
     Statistic GetNumberOfUppercaseChars(size_t index) const;
+    // Returns number of diacritic chars in a string column.
+    // Statistic GetNumberOfDiacriticChars(size_t index) const;
+    // Returns the minimal amount of chars in a column as an int value.
+    Statistic GetMinChars(size_t index) const;
```

Comment:
```suggestion
    Statistic GetMinNumerOfChars(size_t index) const;
```
Same for `GetMaxChars`.